app/views/content/is-teaching-right-for-me/get-school-experience.md

```diff
@@ -60,7 +61,7 @@ $get-school-experience$
 
 ### Teaching internships
 
-A paid [teaching internship](/teaching-internship-providers) could help you to understand what it’s really like in the classroom if you’re currently doing an undergraduate degree and are interested in teaching:
+A paid [teaching internship](/is-teaching-right-for-me/teaching-internship-providers) could help you to understand what it’s really like in the classroom if you’re currently doing an undergraduate degree and are interested in teaching:
```

Comment:
```suggestion
A [paid teaching internship](/is-teaching-right-for-me/teaching-internship-providers) could help you to understand what it’s really like in the classroom if you’re currently doing an undergraduate degree and are interested in teaching:
```
I know this is in prod currently but I wonder if it is worth including `paid` in the link text - we do this elsewhere